pinot-spi/src/main/java/org/apache/pinot/spi/config/table/DedupConfig.java

```diff
@@ -41,4 +48,8 @@ public HashFunction getHashFunction() {
   public boolean isDedupEnabled() {
     return _dedupEnabled;
   }
+
+  public String getKeyStore() {
+    return _keyStore;
```

Comment:
this has a default value?

Reply:
for @JsonProperty, if a default value has not been unspecified it is taken to be empty string

Reply:
right, can you add a comment on the default impl of this store?

Reply:
Will make the change.